src/hooks/useClickOutside/__stories__/useClickOutside.stories.mdx

```diff
@@ -43,7 +43,15 @@ This hook is used when you want to capture click events outside of your componen
     <FunctionArgument
       name="ref"
       type="React.MutableRefObject"
-      description={<>A React <a href="https://reactjs.org/docs/hooks-reference.html#useref" target="_blank" rel="noopener noreferrer">ref</a> object.</>}
+      description={
+        <>
+          A React{" "}
+          <a href="https://reactjs.org/docs/hooks-reference.html#useref" target="_blank" rel="noopener noreferrer">
+            ref
+          </a>{" "}
+          object.
+        </>
+      }
```

Comment:
something feels a bit off with the indentations and the spaces

Reply:
prettier 🤷